### ISSUE TITLE
Details - preview does not follow the blogs anymore

### ISSUE THREAD

**Issue by ErikBrown2** [TumblThreeApp/TumblThree#117]
**Describe the bug**
When you add a few image blogs to the download queue and then in the right pane select Details - Preview, then you will see the images after they are downloaded. With the current versions of TumblThree, when the blog that was being viewed has finished downloading and the next queued blog starts downloading, then the images for that blog will not be automatically shown.

In earlier versions of TumblThree, the images of these next blogs would also be shown automatically. So the Details - Preview display of downloaded images would display the downloaded images for all the queued blogs. But this does not happen anymore in the last few versions of TumblThree.

One other but still related issue with displaying the downloaded images in Details - Preview is that when you select Details - Preview in the right pane and then click on the green highlighted line in the left window, then sometimes the images are not being shown. The images are not shown when you click immediately after the line for a blog becomes green. It appears that you have to wait till the download for that blog actually starts. If you click then on the green line, then the images will be shown. But if you click earlier on the green line, then the images are not shown. You will have to click again on the green line to make the images being shown.

Actually how I expect that it should behave is that at any time when you click on Details - Preview during a download, then it should automatically display the downloaded images. It should not be required to click on the green highlighted line.

**To Reproduce**
1. Go to TumblThree, queue a few blogs for download and start the download by clicking on Crawl.
2. Click on Details - Preview to view the images. Most of the time you will also have to click then on the green highlighted blog on the left pane.
3. At this stage the images will be shown. 
4. To view also images for the next queued blogs you will have to click on the blog entry for these next blogs when they become highlighted in green. This was not necessary with earlier versions of TumblThree.

**Expected behavior**
When you queue a few blogs for download and view them using Details - Preview, then the images for all queued blogs should be shown when they are downloaded. There should be no requirement to click on the next queued blogs.

Also clicking on the green highlighted blog entry immediately after the line becomes green should cause the images to be shown. It should not be required to click again on the line after the images actually started to download.

**Screenshots**

**Desktop (please complete the following information):**
 - TumblThree version: [e.g. 1.0.11.3] 1.0.12.1 but also a few earlier versions.
 - OS: [e.g. Windows 10 Home] Windows 10 Pro
 - Browser [e.g. Chrome, Safari, IE, Edge] Chrome
 - Version [e.g. 22] Version 87.0.4280.88 (Official Build) (64-bit)

**Additional context**

**Comment by thomas694** [comment]
Thanks for your detailed bug report, Erik. To be honest, I'm unaware of this automatic preview following. But I'll have a look at it.
So you mean that behavior changed only recently? Or was it already longer ago?

**Comment by ErikBrown2** [comment]
Thanks for your reply, Thomas.

The feature of preview following the active blog stopped working approximately when you took over the development of TumblThree from johanneszab. The other issue of clicking on the highlighted blog not having any effect till the download for the blog starts was there as long as I used the application, which is over over two years ago.

**Comment by thomas694** [comment]
Well, of course errors could have happened. That's why I've tried a version (v1.0.11.5) from the time before  and took three small blogs without authentication need. Actually it shouldn't be so difficult to add three blogs, put them in the queue and start crawling. But I wasn't able to see that the preview follows the blogs. Maybe I oversee something. Any hints are welcome.

**Comment by ErikBrown2** [comment]
Sorry to see that you took the time to try the old version but were not able to see the following feature. I do not have older versions on my PC so I can not look for a version where it still worked. So then we leave it that, I guess. But since I am here, I would like to use the opportunity to thank you for taking up the continued development of TumblThree. It is a very nice application and it would have been a waste if development had stopped.

**Comment by thomas694** [comment]
Thanks for your words of thanks. I gave back to TumblThree and will do. But hopefully we'll find a very few more contributors (all levels needed and welcome, also "non-developers") in the future.

Well, of course I saw your several contributions from before which I appreciate much. So I thought he knows it longer and remembers better than me and I did my homework and checked it, but couldn't find something that I just had to "re-enable".
But in the meantime I did some work and the "preview following the blogs" should work now. Maybe you like to check it out, now or in the next release. CU and cheers!

All old official versions can be found under ["Releases"](https://github.com/TumblThreeApp/TumblThree/releases) and the current development version which contains all currently in the master repository can be found by clicking on the small green check mark (then Details > Artifacts) besides the commit number on the main page.
